### ISSUE TITLE
The pretrained model

### ISSUE THREAD

**Issue by ShaochengShen** [mightycatty/AIGC_imagery_distortion_detection#1]
Hi！What a wonderful work!
However, I wonder if you have plan to opensource the pretrained ckpt on EvalMuse-40K?
Thanks a lot!

**Comment by mightycatty** [comment]
> Hi！What a wonderful work!
> However, I wonder if you have plan to opensource the pretrained ckpt on EvalMuse-40K?
> Thanks a lot!
> 

Sorry for the late reply. 
This work was specifically trained and evaluated on dataset from nitre2025 workshop. And we didn't pretrained the model on evalmuse 40k.

**Comment by ShaochengShen** [comment]
It was my oversight. Since the NTIRE2025 workshop has two tracks, and the dataset for the other track is EvalMuse-40K, I confused the two.
Btw, I’d like to know if you have any plans to open-source any pre-trained model weights. Because when I tried your code on the NTIRE2025 workshop dataset, I encountered a small bug. I fixed the bug in my own way, but the model's performance wasn’t ideal—it couldn’t output clear map information. I’m not sure if it’s an error on my end. Thanks!

**Comment by mightycatty** [comment]
> It was my oversight. Since the NTIRE2025 workshop has two tracks, and the dataset for the other track is EvalMuse-40K, I confused the two. Btw, I’d like to know if you have any plans to open-source any pre-trained model weights. Because when I tried your code on the NTIRE2025 workshop dataset, I encountered a small bug. I fixed the bug in my own way, but the model's performance wasn’t ideal—it couldn’t output clear map information. I’m not sure if it’s an error on my end. Thanks!

Sorry we don't have plan to opensource the checkpoint due to company IT restrictions. 
As for mask output, did you add dice iou loss? It's critical for performance due to highly unbalanced foreground and background. Using only mse loss tends to output a vague mask in my experience.

**Comment by ShaochengShen** [comment]
> > It was my oversight. Since the NTIRE2025 workshop has two tracks, and the dataset for the other track is EvalMuse-40K, I confused the two. Btw, I’d like to know if you have any plans to open-source any pre-trained model weights. Because when I tried your code on the NTIRE2025 workshop dataset, I encountered a small bug. I fixed the bug in my own way, but the model's performance wasn’t ideal—it couldn’t output clear map information. I’m not sure if it’s an error on my end. Thanks!
> 
> Sorry we don't have plan to opensource the checkpoint due to company IT restrictions. As for mask output, did you add dice iou loss? It's critical for performance due to highly unbalanced foreground and background. Using only mse loss tends to output a vague mask in my experience.

Thanks for your advice!
Btw, I have one more question: since the dataset from NTIRE2025 ws is complicated, have you do any action to the data? (such as pick up the subset of the dataset or add more data)
Sincerely hoping for your reply!

**Comment by mightycatty** [comment]
> > > It was my oversight. Since the NTIRE2025 workshop has two tracks, and the dataset for the other track is EvalMuse-40K, I confused the two. Btw, I’d like to know if you have any plans to open-source any pre-trained model weights. Because when I tried your code on the NTIRE2025 workshop dataset, I encountered a small bug. I fixed the bug in my own way, but the model's performance wasn’t ideal—it couldn’t output clear map information. I’m not sure if it’s an error on my end. Thanks!
> > 
> > 
> > Sorry we don't have plan to opensource the checkpoint due to company IT restrictions. As for mask output, did you add dice iou loss? It's critical for performance due to highly unbalanced foreground and background. Using only mse loss tends to output a vague mask in my experience.
> 
> Thanks for your advice! Btw, I have one more question: since the dataset from NTIRE2025 ws is complicated, have you do any action to the data? (such as pick up the subset of the dataset or add more data) Sincerely hoping for your reply!

We indeed trained a multi-class-classification version of model to filter noisy samples, and add authentic images from flick30k to suppress heatmap false positive. You can find related  implementation in source code.
